CHANGELOG/CHANGELOG-3.6.md

```diff
@@ -16,6 +16,7 @@ See [code changes](https://github.com/etcd-io/etcd/compare/v3.5.0...v3.6.0).
 
 - Add command to generate [shell completion](https://github.com/etcd-io/etcd/pull/13133).
 - When print endpoint status, [show db size in use](https://github.com/etcd-io/etcd/pull/13639)
+- Trim the suffix dot from the target in SRV records returned by DNS lookup, see [code change](https://github.com/etcd-io/etcd/pull/13712).
```

Comment:
@ahrtr thanks and lgtm but a nit, if you are okay, can we please hyperlink `Trim the suffix dot from the SRV target` or `Trim the suffix dot` (something like that)  instead of `code change`. I am good with it one time but this may result in more similar changes in future and that's not so readable. Thanks!

Reply:
@spzala  Thanks for the comment, which makes sense. Updated.